src/panel_gwalker/_gwalker.py

```diff
@@ -84,17 +85,23 @@ class GraphicWalker(ReactComponent):
         Servable `GraphicWalker` object that creates a UI for visual exploration of the input DataFrame.
     """
 
-    object = param.DataFrame(doc="""The data to explore.""")
-    fields = param.List(doc="""Optional fields, i.e. columns, specification.""")
-    appearance = param.Selector(
-        default=None,
-        objects=[None, "media", "dark", "light"],
-        doc="""Dark mode preference: 'media', 'dark', 'light' or None.
-        If None the the appearance is derived from pn.config.theme""",
+    object: pd.DataFrame = param.DataFrame(
+        doc="""The data to explore.
+        Please note that if you update the `object`, then the existing charts will not be deleted."""
     )
-
-    config = param.Dict(
-        doc="""Optional extra Graphic Walker configuration. See the
+    fields: list = param.List(doc="""Optional fields, i.e. columns, specification.""")
+    appearance: Literal["panel", "media", "dark", "light"] = param.Selector(
```

Comment:
I changed this from `None` to `"panel"` because it is more specific. None could mean `null` on the client side - i.e 'media'.

It also gives better meaning when used in a `Select` or `RadioButtonWidget`.